From 3383d8d4df8f96e3928ba4b323505443f8ca4f9c Mon Sep 17 00:00:00 2001
From: Kyota Poeti <kyota.exe@gmail.com>
Date: Sat, 11 Jun 2022 13:00:57 -0400
Subject: [PATCH 1/4] readline tonix port

---
 support/shlib-install | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/support/shlib-install b/support/shlib-install
index 661355d..a61b3a9 100755
--- a/support/shlib-install
+++ b/support/shlib-install
@@ -71,7 +71,7 @@ fi
 # Cygwin installs both a dll (which must go in $BINDIR) and an implicit
 # link library (in $libdir)
 case "$host_os" in
-hpux*|darwin*|macosx*|linux*|solaris2*)
+hpux*|darwin*|macosx*|linux*|solaris2*|*tonix*)
 	if [ -z "$uninstall" ]; then
 		chmod 755 ${INSTALLDIR}/${LIBNAME}
 	fi ;;
@@ -146,7 +146,7 @@ bsdi4*|*gnu*|darwin*|macosx*|netbsd*|mirbsd*)
 	fi
 	;;
 
-solaris2*|aix4.[2-9]*|aix[5-9]*|osf*|irix[56]*|sysv[45]*|dgux*|interix*)
+solaris2*|aix4.[2-9]*|aix[5-9]*|osf*|irix[56]*|sysv[45]*|dgux*|interix*|*tonix*)
 	# libname.so -> libname.so.M
 	${echo} ${RM} ${INSTALLDIR}/$LINK1
 	if [ -z "$uninstall" ]; then
-- 
2.25.1

