From e471116673ce7dd8b13a5d16b0fbe5e92a9700de Mon Sep 17 00:00:00 2001
From: Kyota Poeti <kyota.exe@gmail.com>
Date: Mon, 30 May 2022 19:35:28 -0400
Subject: [PATCH 1/1] figlet tonix port

---
 figlet.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/figlet.c b/figlet.c
index 6daaf4c..b381383 100644
--- a/figlet.c
+++ b/figlet.c
@@ -65,6 +65,7 @@
 #include <ctype.h>
 #include <sys/stat.h>
 #include <fcntl.h>     /* Needed for get_columns */
+#include <termios.h>
 
 #if defined(unix) || defined(__unix__) || defined(__APPLE__)
 #include <unistd.h>
-- 
2.25.1

